server/platform/services/telemetry/telemetry.go

```diff
@@ -1460,26 +1456,34 @@ func (ts *TelemetryService) trackPluginConfig(cfg *model.Config, marketplaceURL
 		}
 	}
 
+	for _, id := range knownPluginIDs {
+		pluginIdStr := fmt.Sprintf("enable_%s", id)
+		_, exists := pluginConfigData[pluginIdStr]
+		if !exists {
+			pluginConfigData[pluginIdStr] = pluginActivated(cfg.PluginSettings.PluginStates, id)
+		}
+	}
 	pluginsEnvironment := ts.srv.GetPluginsEnvironment()
 	if pluginsEnvironment != nil {
 		if plugins, appErr := pluginsEnvironment.Available(); appErr != nil {
 			ts.log.Warn("Unable to add plugin versions to telemetry", mlog.Err(appErr))
 		} else {
-			// If marketplace request failed, use predefined list
-			if marketplacePlugins == nil {
-				for _, id := range knownPluginIDs {
-					pluginConfigData["version_"+id] = pluginVersion(plugins, id)
-				}
-			} else {
+			// If marketplace request didn't fail, go through it
+			if err != nil {
```

Comment:
This `err` is too far removed, contextually, to use it like this. Before, we used either the `knowPluginIDs` /or/ the `marketplacePlugins`, but now it looks like we do both, so can we just always range over `marketplacePlugins`, even if it happens to be empty, and skip this condition altogether?